### PR TITLE
12.1.x Fixes

### PR DESCRIPTION
**Pull request by ryanemerson**
This reverts the changes to the parent pom as that broke the quarkus version alignment.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,19 +4,20 @@
...
     <artifactId>infinispan-quarkus-parent</artifactId>
-    <version>12.1.1-SNAPSHOT</version>
+    <version>12.1.2-SNAPSHOT</version>
     <name>Infinispan Quarkus :: Parent pom</name>
     <packaging>pom</packaging>
 
     <properties>
-        <quarkus.version>1.11.6.Final</quarkus.version>
+        <infinispan.version>${project.version}</infinispan.version>
+        <quarkus.version>1.11.0.Final</quarkus.version>
```
Comment (wburns) [inline pom.xml:20]: Did you mean to downgrade quarkus here?

Comment (ryanemerson) [reply pom.xml]: Yes, I just used `git revert` on the commits that broke the build . I have added a new commit to bump to 1.11.6.Final, let's see what CI says.

Comment (ryanemerson) [reply pom.xml]: Hmm  it seems like upgrading from 1.11.0.Final to any of the patch versions is what causes the logging issues. I have created another PR that retains all of Pedro's changes except the quarkus upgrade https://github.com/infinispan/infinispan-quarkus/pull/65

**Comment by ryanemerson** [comment]
Closing in favour of #65

**Pull request closed without merging**